src/store/farms/reducers.ts

```diff
@@ -17,28 +17,28 @@ import { farmsInitialState, FarmsState } from './state';
 export const farmsReducer = createReducer<FarmsState>(farmsInitialState, builder => {
   builder.addCase(loadSingleFarmStakeActions.submit, state => ({
     ...state,
-    stakesLoading: true
+    lastStakes: createEntity(state.lastStakes.data, true)
   }));
   builder.addCase(loadSingleFarmStakeActions.success, (state, { payload: { stake, farmAddress } }) => {
-    const otherStakes = omit(state.lastStakes, farmAddress);
+    const otherStakes = omit(state.lastStakes.data, farmAddress);
+
+    const lastStakes = isDefined(stake)
+      ? createEntity({
+          ...otherStakes,
+          [farmAddress]: stake
+        })
+      : createEntity({ ...otherStakes });
 
     return {
       ...state,
-      lastStakes: isDefined(stake)
-        ? {
-            ...otherStakes,
-            [farmAddress]: stake
-          }
-        : otherStakes,
-      stakesLoading: false
+      lastStakes
```

Comment:
Using `createEntity` here instead of https://github.com/madfish-solutions/templewallet-mobile/pull/864/files#diff-b06d6a176e1a16e5932549cac45158b4ef58b0fbdf6543d615c669dacf26fd36R26 will remove some duplication